### PR TITLE
Support setting client timeout in config file

### PR DESCRIPTION
**Pull request by Linary**
Implement #8

### CODE REVIEW

File: studio-dist/src/assembly/static/conf/hugegraph-studio.properties [hunk 1]
```diff
@@ -5,6 +5,8 @@ graph.server.host=localhost
 graph.server.port=8080
 graph.name=hugegraph
 
+client.timeout=20
```
Comment (javeme) [inline studio-dist/src/assembly/static/conf/hugegraph-studio.properties]: remove space before " graph.server.port"(line 5)

File: studio-api/src/main/java/com/baidu/hugegraph/studio/config/StudioApiOptions.java [hunk 2]
```diff
@@ -79,6 +79,14 @@ public static StudioApiOptions instance() {
                     "hugegraph"
             );
 
+    public static final ConfigOption<Integer> CLIENT_TIMEOUT =
+            new ConfigOption<>(
+                    "client.timeout",
+                    "The timeout of hugegraph client",
```
Comment (javeme) [inline studio-api/src/main/java/com/baidu/hugegraph/studio/config/StudioApiOptions.java]: The timeout of hugegraph client in seconds

**Review by javeme — DISMISSED** [review]

File: studio-api/src/main/java/com/baidu/hugegraph/studio/config/StudioApiOptions.java [hunk 3]
```diff
@@ -79,6 +79,14 @@ public static StudioApiOptions instance() {
                     "hugegraph"
             );
 
+    public static final ConfigOption<Integer> CLIENT_TIMEOUT =
+            new ConfigOption<>(
+                    "client.timeout",
+                    "The timeout of hugegraph client in seconds",
+                    positiveInt(),
+                    20
```
Comment (javeme) [inline studio-api/src/main/java/com/baidu/hugegraph/studio/config/StudioApiOptions.java]: keep 30s

File: studio-dist/src/assembly/static/conf/hugegraph-studio.properties (hunk 1 above)
Comment (javeme) [inline studio-dist/src/assembly/static/conf/hugegraph-studio.properties]: keep 30s

**Review by javeme — CHANGES_REQUESTED** [review]

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**